examples/meta/src/converter/multidimensional_scaling.sg

```diff
@@ -0,0 +1,32 @@
+CSVFile data_fname("../../data/fm_train_real.dat")
+
+#![create_features]
+RealFeatures features(data_fname)
+#![create_features]
+
+#![calculate_distances_before]
+EuclideanDistance distance_before()
+distance_before.init(features, features)
+#![calculate_distances_before]
+
+#![apply_mds]
+MultidimensionalScaling converter()
+converter.set_target_dim(2)
+converter.set_landmark(False)
+RealFeatures embedding = converter.apply(features)
+#![apply_mds]
+
+#![calculate_distances_after]
+EuclideanDistance distance_after()
+distance_after.init(embedding, embedding)
+#![calculate_distances_after]
+
+#![calculate_distance_matrices]
+RealMatrix distance_matrix_after = distance_after.get_distance_matrix()
```

Comment:
move these to calculate_distances_before and calculate_distances_after

No need for its own section